imagelab/DynaPanel.java

```diff
@@ -69,15 +69,17 @@ public DynaPanel(ImgProvider imp) {
         int width  = imp.getWidth();
         int height = imp.getHeight();
         panelImage = imp.getImage();
-        System.out.println("DynaPanel: panelImage = " + panelImage);
+        System.out.println("Using DynaPanel");
+        //System.out.println("DynaPanel: panelImage = " + panelImage);
     }
 
     /**
      * Draw a given image onto the canvas.
      * @param image the image object to be drawn on the canvas
      */
     public void newImage(Image image) {
-        System.out.println("DynaPanel:draw(i): image = " + image);
+        System.out.println("Using DynaPanel");
```

Comment:
I will remove this output as it was only used in testing to confirm use of new class.